package.json

```diff
@@ -324,6 +324,7 @@
     "inspectpack": "4.1.2",
     "jest": "23.6.0",
     "jest-coffee-preprocessor": "1.0.0",
+    "jest-fetch-mock": "^2.1.2",
```

Comment:
Is there any existing dependency that we can use here instead? [fetch-mock](http://www.wheresrhys.co.uk/fetch-mock/) seems more popular, but curious if there's a way to get the same test coverage without adding a dep.

Reply:
Hopefully addressed by https://github.com/artsy/force/pull/4379/commits/4f5f0a031ef5b37b23d3ea27d82f7af99537efc6